server/app/report.go

```diff
@@ -224,3 +228,15 @@ func (a *UserSurveyApp) generateSurveyMetadataFile(survey *model.Survey, key str
 
 	return filePath, nil
 }
+
+func (a *UserSurveyApp) generateServerMetadataFile(key string) (string, error) {
+	fileDir := path.Join(os.TempDir(), "survey_report", key)
```

Comment:
@harshilsharma63 - is "survey_report" the best name for the content of the metadata file? It is not technically a report, right?

Reply:
@catalintomai @catalintomai this not the file name, its the dir name where I store all survey reports - `/tmp/survey_report/<random_key>/survey_report.zip`. The metadata file name isn't configurable, its hardcoded to be `metadata.yaml` in Ben's PR.